protocol/avdl/keybase1/teams_ui.avdl

```diff
@@ -0,0 +1,5 @@
+@namespace("keybase.1")
+
+protocol teams {
+  void teamCreate(int sessionID, string name);
+}
```

Comment:
Sure, no problem. I think I got inspired by `chatUi` naming but then made it `protocol teams { ...` anyway in avd; file.